.github/PULL_REQUEST/new_config.md

```diff
@@ -0,0 +1,24 @@
+---
+name: New Config
+about: A new cluster config
+---
```

Comment:
This YAML front-matter and filename is for having multiple PR templates I believe. As we currently only have one template (and I can't see that changing for the forseeable future), I'm not sure that we need multiple templates.

What do you think about having this file called `.github/PULL_REQUEST_TEMPLATE.md` instead, without the above? Similar to [nf-core/tools](https://github.com/nf-core/tools/blob/master/.github/PULL_REQUEST_TEMPLATE.md) and most other nf-core repos..

Reply:
I agree there may not be another template anytime soon but I don't think the only PRs are going to new configs. For example a fix for a config #139 or just an update #140. We can definitely add a separate catch all for now similar to [nf-core/tools](https://github.com/nf-core/tools/blob/master/.github/PULL_REQUEST_TEMPLATE.md).

Reply:
I don't really understand sorry - if we're not going to have another template anytime soon then why can we not have `.github/PULL_REQUEST_TEMPLATE.md`?

I agree that there will be plenty of PRs that aren't new configs, but people can just ignore the irrelevant parts of the template. Or if we want different templates for different PR topics then I think we should add them together in one go.